clar2wasm/src/standard/standard.wat

```diff
@@ -1727,6 +1728,69 @@
         )
     )
 
+    (func $buff-to-uint-be (param $offset i32) (param $length i32) (result i64 i64)
+        (local $mask_lo i64) (local $mask_hi i64) (local $double v128)
+        (if (i32.gt_u (local.get $length) (i32.const 16))
+            (then (call $runtime-error (i32.const 5)))
+        )
+
+        (if (i32.eqz (local.get $length))
+            (then (return (i64.const 0) (i64.const 0)))
+        )
+
+        ;; SAFETY: this function works because we already have data in the memory,
+        ;;         otherwise we could have a negative offset.
```

Comment:
I agree, this should be okay, so long as we can be sure that the problem would be caught in a test if we decided to change the layout at some later date.

Reply:
I tend to hope that if someone changes it and the test doesn't work anymore, that someone will figure out the issue fast enough by reading this comment 🤞